cloud-init.pl

```diff
@@ -84,6 +84,25 @@ sub apply_user_data {
       system("sh -c \"$runcmd\"");
     }
   }
+
+  if (defined($data->{write_files})) {
+    foreach my $item (@{ $data->{write_files} }) {
+      open my $fh, ">", $item->{path};
+      printf $fh "%s", $item->{content};
```

Comment:
Superfluous `printf`

```suggestion
      print $fh, $item->{content};
```

Reply:
huh, this one will print `$fh` and `$item->{content}` ?

Reply:
Ok I get it : remove the `,`

Reply:
Sorry, I typoed